lerna.json

```diff
@@ -1,5 +1,10 @@
 {
 	"lerna": "2.9.0",
+	"command": {
+		"bootstrap": {
+			"npmClientArgs": ["--no-package-lock"]
```

Comment:
Do we need it in the case where we have `.npmrc` file with the same flag?

Reply:
Not sure to be honest, but looking at [test/integration/lerna-bootstrap.test.js#L53](https://github.com/lerna/lerna/blob/ba794660e66242bff7bbd04397c5932468d6769a/test/integration/lerna-bootstrap.test.js#L53):
`test("--npm-client npm -- --no-optional extends durable npmClientArgs", async () => {` 

If Lerna is just using the `npm` command then Lerna should honor the contents of the `.npmrc` file I expect

Reply:
Removed in https://github.com/WordPress/packages/commit/56185d5964fc65c955c3b109a7251767618c0123

Reply:
It's still looks good. Seems to use `npm` as it is :)